src/hip_module.cpp

```diff
@@ -282,6 +282,19 @@ hipError_t hipHccModuleLaunchKernel(hipFunction_t f, uint32_t globalWorkSizeX,
         localWorkSizeZ, sharedMemBytes, hStream, kernelParams, extra, startEvent, stopEvent, 0));
 }
 
+hipError_t hipModuleGetGlobal(hipDeviceptr_t* dptr, size_t* bytes,
+                              hipModule_t hmod, const char* name) {
+    if (!dptr || !bytes) return hipErrorInvalidValue;
```

Comment:
Since this code has been moved back to hip source, we should be calling ```HIP_INIT_API(hipModuleGetGlobal, dptr, bytes, hmod, name);``` here.

Reply:
thanks. it's done.